templates/workspaces/base/.env.sample

```diff
@@ -1,26 +1,13 @@
-# These are needed if you are testing at the porter level
-export ID="0df3fbe1-38f3-4f2e-9dee-3273351d9b1a"
-export ADDRESS_SPACE="10.2.8.0/24"
-export SHARED_STORAGE_QUOTA=50
-export KEYVAULT_PURGE_PROTECTION_ENABLED=false
-export ENABLE_LOCAL_DEBUGGING=true
-export REGISTER_AAD_APPLICATION=true
-export CLIENT_ID=$WORKSPACE_APP_CLIENT_ID
-
-# These are needed if you are testing at the Terraform level.
-export TRE_ID=__CHANGE_ME__
-export TF_VAR_tre_id=__CHANGE_ME__
-export TF_VAR_location=westeurope
-export TF_VAR_tre_resource_id=0df3fbe1-38f3-4f2e-9dff-3273351d9b1a
-export TF_VAR_address_space="10.2.6.0/24"
-export TF_VAR_enable_local_debugging="true"
-export TF_VAR_purge_protection_enabled="false"
-export TF_VAR_register_aad_application="true"
-export TF_VAR_client_id=$WORKSPACE_APP_CLIENT_ID
-
-export TF_VAR_mgmt_resource_group_name=__CHANGE_ME__
-export TF_VAR_mgmt_storage_account_name=__CHANGE_ME__
-export TF_VAR_terraform_state_container_name=tfstate
-export TF_VAR_auth_tenant_id="__CHANGE_ME__"
-export TF_VAR_auth_client_id="__CHANGE_ME__"
-export TF_VAR_auth_client_secret="__CHANGE_ME__"
+TRE_RESOURCE_ID="__CHANGE_ME__"
```

Comment:
is this a general tidy up of this file?

if so, can we put some space between "stuff you need to change/define" and "stuff that has a default value", and maybe a comment above each?